src/storybook/webpack.d.ts

```diff
@@ -0,0 +1,5 @@
+declare interface Window {
+  __SALEOR_CONFIG__: {
```

Comment:
```suggestion
  __SALEOR_CONFIG__?: {
   APP_MOUNT_URI: string;
  };
}
```

Reply:
why optional? can you run the APP without config?

Reply:
maybe it is not need to have this file at all if it is only for storybook

Reply:
Actually it is not needed for more then storybook and even the file where we define config is in javascript, as well as storybook version is quite old. I removed this file.